unitary/examples/quantum_rpg/xp_utils.py

```diff
@@ -47,6 +47,18 @@ def choose(self) -> List[alpha.QuantumEffect]:
         return list(random.choices(self.xp_choices, weights=self.weights)[0])
 
 
+def is_ready_for_next_level(qar: qaracter.Qaracter) -> bool:
```

Comment:
Can you add a docstring with the same explanation as the PR description? I was confused what was happening here until I read that. Maybe also print some indication to the player on the criteria for leveling up.